app/main.py

```diff
@@ -1,3 +1,74 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None :
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance((self.km + other.km))
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        return print("error")
+
+    def __iadd__(self, other: Distance | int | float) -> Distance | int | float:
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+            return self
+        elif isinstance(other, (int, float)):
+            self.km = self.km + other
+            return self
+        return print("error")
+
+    def __mul__(self, other: Distance | int | float) -> Distance | int | float:
+        if isinstance(other, (int, float)):
+            return Distance(self.km * other)
+        return print("error")
+
+    def __truediv__(self, other: Distance | int | float) -> Distance | int | float:
+        if isinstance(other, (int, float)):
+            return Distance(round((self.km / other), 2))
+        return print("error")
+
+    def __lt__(self, other: Distance | int | float) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif isinstance(other, (int, float)):
+            return self.km < other
+        return print("error")
+
+    def __gt__(self, other: Distance | int | float) -> bool:
+        if isinstance(other, Distance):
+            return self.km > other.km
+        elif isinstance(other, (int, float)):
+            return self.km > other
+        return print("error")
+
+    def __eq__(self, other: Distance | int | float) -> bool:
+        if isinstance(other, Distance):
+            return self.km == other.km
+        elif isinstance(other, (int, float)):
+            return self.km == other
+        return print("error")
+
+    def __le__(self, other: Distance | int | float) -> bool:
```

Comment:
You can use previously created magic method here(and in ge as well) to simplify your code. e.g. return not self > other